app/presenters/variant_tsv_presenter.rb

```diff
@@ -29,7 +29,8 @@ def self.headers
       'variant_types',
       'hgvs_expressions',
       'last_review_date',
-      'civic_actionability_score'
+      'civic_actionability_score',
+      'allele_registry_id',
```

Comment:
Generally you wouldn't want trailing commas but it doesn't really hurt anything either.

Reply:
It's a stylistic choice but I prefer to add them so next time someone appends a field to the list, the diff only shows the new field and not the comma addition to the previous line.